test-framework/core/src/main/java/org/keycloak/testframework/server/KeycloakServerConfigBuilder.java

```diff
@@ -147,22 +147,12 @@ private void build() {
                 option("log-console-format", format);
             }
 
-            StringBuilder logLevel = new StringBuilder();
             if (rootLevel != null) {
-                logLevel.append(rootLevel);
+                option("log-level", rootLevel);
             }
 
             for (Map.Entry<String, String> e : categoryLevels.entrySet()) {
-                if (!logLevel.isEmpty()) {
```

Comment:
@miquelsi is is causing some errors?

Reply:
Was planning to use these new options when they where ready anyways, so this is a good change, and not just a workaround ;)